### PR TITLE
style(home): Add hover effect on card and wrap entire block in NavLink

### PR DESCRIPTION
**Pull request by noblica**
Wrapped entire blocks into the `<NavLink>` component, so it's entirely clickable.
Also added a hover effect, to make it clearer that the element is clickable.

Here's how it looks now:

https://user-images.githubusercontent.com/9339019/116486644-61872000-a88e-11eb-9b28-f053f0023ae5.mp4

Please let me know if the hover effect is fine, or if further changes are needed :slightly_smiling_face:

### CODE REVIEW

**Comment by ChefBugs** [comment]
Thanks! The hover effect is a bit aggressive though, for now please let's use the same hover effect as the buttons https://pancakeswap.github.io/pancake-uikit/?path=/story/components-button--default

**Comment by noblica** [comment]
Hey @ChefBugs , thanks for the feedback.
I updated the PR so now the hover effect looks like this:

https://user-images.githubusercontent.com/9339019/116527577-1c3d0f80-a8db-11eb-933a-3300261573cf.mp4


Please let me know if more changes are needed :slightly_smiling_face:

File: src/views/Home/components/EarnAPRCard.tsx [hunk 1]
```diff
@@ -18,6 +18,12 @@ const StyledFarmStakingCard = styled(Card)`
     margin: 0;
     max-width: none;
   }
+
+  transition: opacity 200ms;
+  &:hover {
+    opacity: 0.65;
+    transition: opacity 200ms;
```
Comment (hachiojidev) [inline src/views/Home/components/EarnAPRCard.tsx:25]: No need to add the `transition` property on this line.

File: src/views/Home/components/EarnAssetCard.tsx [hunk 2]
```diff
@@ -15,6 +15,12 @@ const StyledFarmStakingCard = styled(Card)`
     margin: 0;
     max-width: none;
   }
+
+  transition: opacity 200ms;
+  &:hover {
+    opacity: 0.65;
+    transition: opacity 200ms;
```
Comment (hachiojidev) [inline src/views/Home/components/EarnAssetCard.tsx:22]: Same here

File: src/views/Home/components/WinCard.tsx [hunk 3]
```diff
@@ -13,6 +13,12 @@ const StyledFarmStakingCard = styled(Card)`
     margin: 0;
     max-width: none;
   }
+
+  transition: opacity 200ms;
+  &:hover {
+    opacity: 0.65;
+    transition: opacity 200ms;
```
Comment (hachiojidev) [inline src/views/Home/components/WinCard.tsx:20]: And here...

**Review by hachiojidev — CHANGES_REQUESTED** [review]

**Comment by hachiojidev** [comment]
@noblica Please resolve conflicts

**Comment by noblica** [comment]
@hachiojidev Done :+1:

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**